knowage-vue/src/modules/managers/mondrianSchemasManagement/MondrianSchemasManagementTabView.spec.js

```diff
@@ -0,0 +1,70 @@
+import { mount } from '@vue/test-utils'
+import axios from 'axios'
+import Button from 'primevue/button'
+import Column from 'primevue/column'
+import Card from 'primevue/card'
+import DataTable from 'primevue/datatable'
+import flushPromises from 'flush-promises'
+import InputText from 'primevue/inputtext'
+import KnValidationMessages from '@/components/UI/KnValidatonMessages.vue'
+import MondrianSchemasManagementTabView from './MondrianSchemasManagementTabView.vue'
+import ProgressBar from 'primevue/progressbar'
+import TabPanel from 'primevue/tabpanel'
+import TabView from 'primevue/tabview'
+import Toolbar from 'primevue/toolbar'
+
+const $store = {
+    commit: jest.fn()
+}
+
+const $router = {
+    replace: jest.fn()
+}
+
+const factory = () => {
+    return mount(MondrianSchemasManagementTabView, {
+        global: {
+            stubs: {
+                Button,
+                Column,
+                Card,
+                DataTable,
+                KnValidationMessages,
+                InputText,
+                ProgressBar,
+                TabPanel,
+                TabView,
+                Toolbar
+            },
+            mocks: {
+                $t: (msg) => msg,
+                $store,
+                $router
+            }
+        }
+    })
+}
+
+describe('Mondrian Schema Management Tab View', () => {
+    it('switches to Workflow tab if Workflow  is clicked', async () => {
+        const wrapper = factory()
+
+        await flushPromises()
+        await wrapper.find('.p-tabview-nav li:nth-child(1)').trigger('click')
+
+        console.log('--------------- NAV 1 ------------------')
```

Comment:
Remember to remove console logs when committing

Reply:
Apologies, one log escaped me, should i remove it now or are there any additional changes to be made?

Reply:
Everything else seems fine code-side. If you can just remove it will be perfect thx!